index.html

```diff
@@ -41,25 +43,21 @@ <h2>My Three Favorite ball games Basketball, football and tennis</h2>
             </ol>
         </section>
         <section class="bio">
-            <span style="background: rgb(255, 207, 102);display: block; padding: 1px;padding-left: 15px">
```

Comment:
This looks like a stray bit of code from you experimenting.  Do your experimenting in a separate project from the lab